src/createServer.js

```diff
@@ -1,11 +1,24 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
+const cors = require('cors');
+
+const usersRouter = require('./routes/usersRouter');
+const expensesRouter = require('./routes/expensesRouter');
+const usersService = require('./services/usersService');
+const expensesService = require('./services/expensesService');
 
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
+  usersService.getInitial();
+  expensesService.getInitial();
+
+  const app = express();
+
+  app.use(cors());
+  app.use('/users', express.json(), usersRouter);
+  app.use('/expenses', express.json(), expensesRouter);
```

Comment:
A little simpler this way :)

```suggestion

  app.use(cors());
  app.use(express.json());
  app.use('/users', usersRouter);
  app.use('/expenses', expensesRouter);
```